cli.js

```diff
@@ -80,7 +80,7 @@ var cli = meow([
 	'  ava --init foo.js',
 	'',
 	'Default patterns when no arguments:',
-	'test.js test-*.js test/**/*.js'
+	'test.js test-*.js test/**/*.js __tests__/**/*.js **/*.test.js'
```

Comment:
I thought the convention is that `__tests__` can be anywhere? Hence why it's wrapped in underscores.